docker-stack.yml

```diff
@@ -0,0 +1,15 @@
+version: '3'
+services:
+  blaze-server:
+    image: node:10.16.3
```

Comment:
Why are we using the standard node image here? Blaze has its own server and client images.

Reply:
👍 I'll change it